### PR TITLE
various: Reset default mirror to repo.voidlinux.org

### PR DESCRIPTION
**Pull request by the-maldridge**
I'd like to retire the alpha.de.repo.voidlinux.org name.  The first step of this is removing it from documentation and active automated use.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Don't request update of package. We have a script for that. https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt . However, a quality pull request may help. -->
+<!-- Don't request update of package. We have a script for that. https://repo-fi.voidlinux.org/void-updates/void-updates.txt . However, a quality pull request may help. -->
```
Comment (classabbyamp) [inline .github/issue_template.md:1]:
```
$ curl -s https://repo-fi.voidlinux.org/void-updates/void-updates.txt | head -n 1
Void Updates for 2021-09-26 04:11 UTC (took: 678s)
```
this is a bit out of date

**Comment by Chocimier** [comment]
What is planned time window from pushing update of xbps and void-*-repos to shutting down domain? I would argue for 9-12 months, as usually.

**Comment by the-maldridge** [comment]
I had no particular time window in place.  I was largely going to handle this the same way as when repo.voidlinux.org was shut off where the domain was left on basically until I remembered almost a year later that it could be turned off.  I might be more aggressive with turning off alpha.us.repo.voidlinux.org though as that was never a default mirror and hasn't appeared in documentation for a long time.

**Comment by 0x5c** [comment]
Would an option be to setup a "generic" subdomain that points to whatever official repo should be the default? Then next time a default repo change is needed, it would only be a matter of changing where it points to.

**Comment by Anachron** [comment]
While we're at it we could then automatically route to fastest mirror based on user location... ;-)

**Comment by the-maldridge** [comment]
I'm still not super happy with this.  Right now it uses repo.voidlinux.org, but I'm considering repo-default.voidlinux.org instead since I think that makes it clearer that this is a default repo that you're using rather than a perhaps more optimized one.  What do people think about this?

**Comment by classabbyamp** [comment]
I think repo-default is more clear :+1:

**Comment by 0x5c** [comment]
I think it's clearer too, and also suggest that the actual server location could change.

**Pull request closed without merging**